react-app-drawer-component/src/app-drawer.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+export default class AppDrawer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { isSatisfied: false };
+    this.handleClick = this.handleClick.bind(this);
+    this.modalOff = this.modalOff.bind(this);
+    this.closeModal = this.closeModal.bind(this);
+    this.handleClickWhenOn = this.handleClickWhenOn.bind(this);
+  }
+
+  handleClick() {
+    if (event.target.className === 'fas fa-bars fa-2x') {
+      this.setState(prevState => ({
+        isSatisfied: !prevState.isSatisfied
+      }));
+    }
+  }
+
+  handleClickWhenOn() {
```

Comment:
If you are going to use `event` in your event handler callback make sure to use the `event` parameter.

```JAVASCRIPT
handleClickWhenOn(event) {
```